packages/frontend/src/features/sqlRunner/components/CreateVirtualViewModal.tsx

```diff
@@ -117,6 +120,7 @@ export const CreateVirtualViewModal: FC<Props> = ({ opened, onClose }) => {
                         required
                         // TODO - don't allow duplicate names
                         {...form.getInputProps('name')}
+                        error={!!error?.error}
```

Comment:
I tried to do a `instaceof AlreadyExistsError` but it was always returning `false`, so I've only enabled the red border (we display the error message on the toast instead)